### PR TITLE
docs: migrate from postgresql

### PR DESCRIPTION
**Pull request by MichaelScofield**
## What's Changed in this PR

as title

## Checklist

- [x] Please confirm that all corresponding versions of the documents have been revised.
- [x] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+If retaining all historical data isn't necessary, you can simultaneously write data to both GreptimeDB and PostgreSQL
+for a specific period to accumulate the required recent data. Subsequently, cease writing to PostgreSQL and continue
+exclusively with GreptimeDB. If a complete migration of all historical data is needed, please proceed with the following
+steps.
+
+### Export data from PostgreSQL
+
+[pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) is a commonly used tool to export data from
+PostgreSQL. Using it, we can export the data that can be later imported into GreptimeDB directly. For example, if we
+want to export two schemas, `db1` and `db2` in the database `postgres` from PostgreSQL, we can use the following
```
Comment (nicecui) [inline docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:60]:
```suggestion
want to export schemas whose names start with `db` in the database `postgres` from PostgreSQL, we can use the following
```

File: docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+command:
+
+```bash
+pg_dump -h127.0.0.1 -p5432 -Upostgres -ax --column-inserts --no-comments -n 'db*' postgres | grep -v "^SE" > /path/to/output.sql
+```
+
+Replace the `-h`, `-p` and `-U` flags with the appropriate values for your PostgreSQL server. The `-n` flag is used to
+specify the schemas to be exported. Add the database in the end. Note that we pipe the pg_dump output through a special
```
Comment (nicecui) [inline docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:68]:
```suggestion
specify the schemas to be exported. And the database `postgres` is at the end of the `pg_dump` command line. Note that we pipe the `pg_dump` output through a special
```

File: docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+`grep`, to remove some unnecessary "`SET`" or "`SELECT`" lines. The final output will be written to the
```
Comment (nicecui) [inline docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:69]:
```suggestion
`grep`, to remove some unnecessary `SET` or `SELECT` lines. The final output will be written to the
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+GreptimeDB，请参考[写入数据](/user-guide/ingest-data/for-iot/sql.md)部分。
+
+若无需保留所有历史数据，你可以双写一段时间以积累业务所需的最新数据，然后停止向 PostgreSQL 写入数据并仅使用
+GreptimeDB。如果需要完整迁移所有历史数据，请按照接下来的步骤操作。
+
+### 从 PostgreSQL 导出数据
+
+[pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) 是一个常用的、从 PostgreSQL 导出数据的工具。使用
+pg_dump，我们可以从 PostgreSQL 中导出后续可直接导入到 GreptimeDB 的数据。例如，如果我们想要从 PostgreSQL 的 database
+`postgres` 中导出两个 schema `db1` 和 `db2`，我们可以使用以下命令：
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:44]:
```suggestion
`postgres` 中导出以 `db` 开头的 schema，我们可以使用以下命令：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+```bash
+pg_dump -h127.0.0.1 -p5432 -Upostgres -ax --column-inserts -n 'db*' postgres | grep -v "^SE" > /path/to/output.sql
+```
+
+替换 `-h`、`-p` 和 `-U` 参数为 PostgreSQL 服务的正确值。`-n` 参数用于指定要导出的 schema。将 database 放在命令最后。注意这里我们将
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:50]:
```suggestion
替换 `-h`、`-p` 和 `-U` 参数为 PostgreSQL 服务的正确值。`-n` 参数用于指定要导出的 schema。数据库 `postgres` 被放在了 `pg_dump` 命令的最后。注意这里我们将
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+pg_dump 的输出经过了一个特殊的 `grep` 命令以过滤掉不需要的 "`SET`" 和 "`SELECT`" 行。最终输出将写入 `/path/to/output.sql`
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:51]:
```suggestion
pg_dump 的输出经过了一个特殊的 `grep` 命令以过滤掉不需要的 `SET` 和 `SELECT` 行。最终输出将写入 `/path/to/output.sql`
```

File: i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+”[psql -- PostgreSQL interactive terminal](https://www.postgresql.org/docs/current/app-psql.html)“ 可用于将数据导入
+GreptimeDB。继续上面的示例，假设数据导出到文件 `/path/to/output.sql`，那么我们可以使用以下命令将数据导入 GreptimeDB：
+
+```bash
+psql -h127.0.0.1 -p4003 -d public -f /path/to/output.sql
+```
+
+替换 `-h` 和 `-p` 参数为你的 GreptimeDB 服务的值。`-d` 参数是为 psql 指定 database。为使 psql 能连接上 GreptimeDB，`-d`
+参数应固定为 "public"。添加 `-a` 以查看详细的执行结果，或使用 `-s` 进入单步执行模式。
```
Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:102]:
```suggestion
参数应固定为 `public`。添加 `-a` 以查看详细的执行结果，或使用 `-s` 进入单步执行模式。
```

Comment (nicecui) [inline i18n/zh/docusaurus-plugin-content-docs/current/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:102]:
```suggestion
替换 `-h` 和 `-p` 参数为你的 GreptimeDB 服务的值。psql 命令中的 `-d` 参数用于指定数据库，该参数不能省略，`-d` 的值 `public` 是 GreptimeDB 默认使用的数据库。你还可以添加 `-a` 以查看详细的执行结果，或使用 `-s` 进入单步执行模式。
```

File: docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+The [psql -- PostgreSQL interactive terminal](https://www.postgresql.org/docs/current/app-psql.html) can be used to
+import data into GreptimeDB. Continuing the above example, say the data is exported to file `/path/to/output.sql`, then
+we can use the following command to import the data into GreptimeDB:
+
+```bash
+psql -h127.0.0.1 -p4003 -d public -f /path/to/output.sql
+```
+
+Replace the `-h` and `-p` flags with the appropriate values for your GreptimeDB server. The `-d` flag is the database to
+use for psql. For psql to connecting to GreptimeDB, it should be fixed to "public". Add `-a` to the end to see every
```
Comment (nicecui) [inline docs/user-guide/migrate-to-greptimedb/migrate-from-postgresql.md:121]:
```suggestion
Replace the `-h` and `-p` flags with the appropriate values for your GreptimeDB server. The `-d` of the psql command is used to specify the database and cannot be omitted. The value `public` of `-d` is the default database used by GreptimeDB. You can also add `-a` to the end to see every
```

**Review by nicecui — COMMENTED** [review]
LGTM

**Review by nicecui — APPROVED** [review]

**Comment by nicecui** [comment]
could you please also add the PostgreSQL content to v0.9? thank you. I'll create another PR to MySQL

**Comment by MichaelScofield** [comment]
@nicecui done

**Review by fengjiachun — APPROVED** [review]

**Pull request merged**